src/models/userDao.js

```diff
@@ -89,26 +89,36 @@ const editUserInfo = async (userId, userName, cellphone, sex, bio) => {
   }
 };
 
-const checkUserNameExists = async (userName) => {
+const uploadImageUrl = async (userId, profileImageUrl) => {
   try {
-    const [user] = await dataSource.query(
-      `
-      SELECT id
-      FROM users
-      WHERE user_name = ?
-      `,
-      [userName]
-    );
-    return !!user; // Convert the result to a boolean value
+    if (profileImageUrl) {
+      await dataSource.query(
+        `UPDATE users
+        SET
+          profile_image_url = ?
+        WHERE id = ?`,
+        [profileImageUrl, userId]
+      );
+    } else {
+      await dataSource.query(
+        `UPDATE users
+        SET
+          profile_image_url = NULL
+        WHERE id = ?`,
+        [userId]
+      );
+    }
```

Comment:
어차피 profileImage가 없을 때 null을 넣고 싶은 거면  if 문을 안 사용해도 될 것 같습니다.
```suggestion
     await dataSource.query(
        `UPDATE users
        SET
          profile_image_url = ?
        WHERE id = ?`,
        [profileImageUrl, userId]
      );
```